graph.js

```diff
@@ -638,13 +653,25 @@ class Graph {
         }
     }
 
-    load(nodes, edges) {
+    loadDocuments(nodes) {
         this.nodeId = nodes.reduce(function (prev, curr) {
             return (prev.id > curr.id) ? prev.id : curr.id
         });
-        this.nodes = nodes;
+        this.nodes = this.nodes.concat(nodes)
+        this.updateNodes("document");
+    }
+
+    loadEntities(nodes) {
+        this.nodeId = nodes.reduce(function (prev, curr) {
+            return (prev.id > curr.id) ? prev.id : curr.id
+        });
```

Comment:
I wonder if there's a nicer way to get the max id. I think this logic can be placed in a function `getMaxId` for clarity, and perhaps replace line 666 with `return Math.max(prev.id, curr.id);`
The class variable should also be renamed to `maxNodeId`.

Reply:
tbh i havent thought about this either, this was just copy pasta from the original repo. yeap max would be more readable!